src/client/src/orangehrmBuzzPlugin/components/NewsFeed.vue

```diff
@@ -39,7 +39,9 @@
           :posted-date="post.createdTime"
         >
           <template #content>
-            <oxd-text>{{ post.text }}</oxd-text>
+            <oxd-text v-if="post.text" tag="p">
+              {{ post.text }} {{ post.type }}
```

Comment:
```vue
              {{ post.type }}
```
:thinking: do we need this?

Reply:
This is for QA to see the type of post when testing share post function